src/core/lib/iomgr/ev_epoll1_linux.cc

```diff
@@ -1228,6 +1284,7 @@ static const grpc_event_engine_vtable vtable = {
     pollset_set_add_fd,
     pollset_set_del_fd,
 
+    workqueue_scheduler,
```

Comment:
This really needs to be fd_workqueue_scheduler: we need some entity against which to schedule, and for chttp2 (where we're focussing) this is an fd... 

other pollers do have some kind of work sharding and we'll switch from epoll1 soon enough

Reply:
This was just a POC so I could get some experimental numbers.  In the case of epoll1 it looks like we would ignore the fd anyways.

I didn't see much improvement (0.841 Meta-score).